docs/resources/configure_cluster.plantuml

```diff
@@ -0,0 +1,25 @@
+@startuml
+title Configure OSD cluster to be used by OSIO
+
+actor "OSD admin" as admin
+box "OSD Cluster" #LightBlue
+	participant "OSD cluster console\nlocal marketplace" as marketplace
+	participant "OSIO operator" as operator
+	participant "cluster API" as cluster
+end box
+box "OSIO"
+	participant "Cluster Service" as cs
+	participant "Auth Service" as auth
+end box
+
+admin -> marketplace : install OSIO operator
+marketplace --> operator
+operator -> cluster : create OAuthClient
+operator <-- cluster : OAuthClient credentials
+operator <-- cluster : SA token
+|||
+operator -> auth : obtain OSIO operator Service Account token
+operator <-- auth : {operator_sa_token}
```

Comment:
Can you plz add here, what auth_mechanism allow auth_service to return operator_sa_token to osio_operator ??

Reply:
It's https://fabric8-services.github.io/fabric8-auth/reference.html#_service_account_authentication
This flow is a very first draft. We will add more details later.